src/modules/dashboard/components/Dialogs/EditExpenditureDialog/EditExpenditureDialogForm.tsx

```diff
@@ -0,0 +1,320 @@
+import React, { useCallback, useMemo } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import classNames from 'classnames';
+import { FormikProps } from 'formik';
+import { nanoid } from 'nanoid';
+import { ROOT_DOMAIN_ID } from '@colony/colony-js';
+
+import { Annotations, FormSection, Toggle } from '~core/Fields';
+import { DialogSection } from '~core/Dialog';
+import MotionDomainSelect from '~dashboard/MotionDomainSelect';
+import Heading from '~core/Heading';
+import { ValuesType } from '~pages/ExpenditurePage/ExpenditurePage';
+import Button from '~core/Button';
+import ColorTag, { Color } from '~core/ColorTag';
+import { getAllUserRoles } from '~modules/transformers';
+import { hasRoot } from '~modules/users/checks';
+import { useDialogActionPermissions } from '~utils/hooks/useDialogActionPermissions';
+import { useTransformer } from '~utils/hooks';
+import { Colony, useLoggedInUser } from '~data/index';
+
+import { FormValuesType } from './EditExpenditureDialog';
+import Recipient from './Recipient';
+import styles from './EditExpenditureDialogForm.css';
+
+export const MSG = defineMessages({
+  header: {
+    id: 'dashboard.EditExpenditureDialog.EditExpenditureDialogForm.header',
+    defaultMessage: 'Create a motion to edit payment',
+  },
+  descriptionText: {
+    id: `dashboard.EditExpenditureDialog.EditExpenditureDialogForm.descriptionText`,
+    defaultMessage: `Payment is currently at the locked stage.
```

Comment:
Please change this text to be: "This Payment is currently Locked. Either a Motion, or a member with the Arbitration permission are required to make changes. "